README.md

```diff
@@ -1 +1 @@
-# testRepo
\ No newline at end of file
+# Create folder “Task1” and add a file to the folder (branch “testBranch”), create a PR in Github and add lecture instructor as a reviewer.
```

Comment:
Please rewrite to more correct one, in case if you added README.md.
Main point of README: to describe what purpose of this repository

Reply:
Or add KH-067-QA-02-FirstName-LastName(your)

Reply:
Done. I deleted 2 files and text in the firstFile in the folder "Task1". And I updated the readme file.